src/Linear/V0.hs

```diff
@@ -14,6 +14,10 @@
 {-# LANGUAGE DataKinds #-}
 #endif
 
+#if __GLASGOW_HASKELL__ >= 801
```

Comment:
Surely this should be `800`, not `801`?

Reply:
Sure! “Since `8.0.1`”...

I always need to turn around the CPP version numbers in my brain before I get it right.